cli/config/config_test.go

```diff
@@ -150,9 +150,8 @@ func TestOldValidAuth(t *testing.T) {
 
 	// defaultIndexserver is https://index.docker.io/v1/
 	ac := config.AuthConfigs["https://index.docker.io/v1/"]
-	if ac.Username != "joejoe" || ac.Password != "hello" {
-		t.Fatalf("Missing data from parsing:\n%q", config)
-	}
+	assert.Equal(t, ac.Username, "joejoe")
+	assert.Equal(t, ac.Password, "hello")
```

Comment:
Aren't we losing information here in case of a failure?. Previously we would have been able to see the config that fails the test (and the password with it :P)

Reply:
@tiborvass right, but that's not what we check at all :angel:. We can still add a comment to these assert with the `config` if we want :wink: